multinode/roles/grafana/tasks/main.yml

```
@@ -0,0 +1,186 @@
+        enabled: yes
+
+- name: Install Grafana Loki
+  block:
+    - name: Download the Loki zip file
+      get_url:
+        url: https://github.com/grafana/loki/releases/download/v3.2.0/loki-linux-amd64.zip
+        dest: "{{ user_home }}/loki-linux-amd64.zip"
+
+    - name: sleep for a few seconds
+      pause:
+        seconds: 30    
```

yamllint

[error] trailing spaces

(trailing-spaces)